team.html

```diff
@@ -0,0 +1,341 @@
+<!DOCTYPE html>
+<html lang="en">
+	<!-- This file is a placehodler for a page. Rename this file to the name of the page you are working on. -->
+
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>If These Lands Could Talk - Page Name</title>
+		<link
+			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link rel="stylesheet" href="css/global.css" />
+		<!-- In a new branch, rename this page.css file to the name of that actual page (Example: home.css) -->
+		<link rel="stylesheet" href="css/team.css" />
+	</head>
+
+	<body>
+		<div class="header-section black-text">
+			<div class="header-content">
+				<div class="header-picture">
+					<!-- add soil hands img -->
+					<picture>
+						<source
+							media="(min-width:768px)"
+							srcset="./assets/images/pages/our-team/desktop/earth-d.jpg"
+						/>
+						<img
+							src="./assets/images/pages/our-team/earth.jpg"
+							alt="two sets of hands, one holding a tiny plant with soil"
+							style="width: auto"
+						/>
+					</picture>
+				</div>
+				<div class="header-info wrapper">
+					<h3>Our Team</h3>
+					<p>
+						The vision of our team is to leverage digital technology and
+						innovation to improve STEAM (Science, Technology, Engineering, Art,
+						and Math) excellence, retention, community building, and advancement
+						in native and indigenous communities across the world.
+					</p>
+				</div>
+			</div>
+		</div>
+		<div class="team-parent">
```

Comment:
Looks good visually, I am wondering if you can use a global **.wrapper** class to help the mobile --> desktop spacing. the **team-parent** and other section styles look good, but i think leveraging the **.wrapper ** class might result in less css!

Reply:
Here is a screen shot for reference!

![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/98284615/f040bd40-c6dc-486a-aa8e-6f78eb6c7a5c)